pkg/parser/model/ddl.go

```diff
@@ -791,7 +841,7 @@ func (job *Job) String() string {
 		warnings, _ := job.GetWarnings()
 		ret += fmt.Sprintf(", UniqueWarnings:%d", len(warnings))
 	}
-	if job.Type != ActionMultiSchemaChange && job.MultiSchemaInfo != nil {
+	if job.Type == ActionMultiSchemaChange && job.MultiSchemaInfo != nil {
```

Comment:
Why do you change it?

Reply:
Could there be `job.MultiSchemaInfo != nil` if `job.Type` isn't `ActionMultiSchemaChange`?